frontend/packages/knative-plugin/src/topology/listView/NoStatusListViewNode.tsx

```diff
@@ -1,15 +1,29 @@
 import * as React from 'react';
-import { TopologyListViewNode } from '@console/topology/src/components/list-view';
+import { SignOutAltIcon } from '@patternfly/react-icons';
+import {
+  TopologyListViewNode,
+  TypedResourceBadgeCell,
+} from '@console/topology/src/components/list-view';
 import { OdcBaseNode } from '@console/topology/src/elements';
+import { getResourceKind } from '@console/topology/src/utils';
 
 interface NoStatusListViewNodeProps {
   item: OdcBaseNode;
   selectedIds: string[];
   onSelect: (ids: string[]) => void;
 }
 
-const NoStatusListViewNode: React.FC<NoStatusListViewNodeProps> = (props) => (
-  <TopologyListViewNode noPods {...props} />
-);
-
+const NoStatusListViewNode: React.FC<NoStatusListViewNodeProps> = (props) => {
+  const kind = getResourceKind(props.item);
+  const badgeCell = (
+    <TypedResourceBadgeCell key="type-icon" kind={kind} typeIcon={<SignOutAltIcon />} />
+  );
+  return (
+    <TopologyListViewNode
+      noPods
+      {...props}
+      badgeCell={props.item.getType() === 'event-source' ? badgeCell : null}
```

Comment:
Can we make use of `NodeType` enum instead of hardcoding `event-source`?
```suggestion
      badgeCell={props.item.getType() === NodeType.EventSource ? badgeCell : null}
```